### DISCUSSION TITLE
Welcome to DxSvelte Discussions

### DISCUSSION THREAD

**Discussion by emmyarty** [emmyarty/dxsvelte#3] [Announcements]
## 🤍 Well hello there!
  These Discussions are a place to connect with other users and contributors. You're invited to:
  * Ask questions you’re wondering about.
  * Share ideas and feedback.
  * Discuss ideas for new features or existing feature implementations.
  * Engage with other community members.
  * Welcome others and behave positively and with an open mind. Remember that we're doing this together.

  Introduce yourself in a comment below if you feel inclined, and tell us a bit about yourself!

**Comment by RustyDimes** [comment]
Hello, i've some silly questions about a possible cooperation/contribution mechanism. Check your email inbox, please.
